.github/workflows/dependent-issues.yml

```diff
@@ -0,0 +1,36 @@
+name: Dependent Issues
+
+on:
+  issues:
+    types:
+      - opened
+      - edited
+      - reopened
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+  schedule:
+    - cron: '15 * * * *' # schedule hourly check for external dependencies
```

Comment:
What's the scenario where this is needed?

Reply:
One of our PRs might depend on a issue / PR in an external repo (e.g. community Lean) so this will make sure those are checked once an hour. I was hoping the comment would make this clear, so let me know if you can think of a way to improve it.

Reply:
Ah, I just realized that this also manages the PRs which were opened before this was merged.